ClearKeep/Scenes/Settings/SettingServerView.swift

```diff
@@ -111,6 +111,8 @@ struct SettingServerView: View {
 			}
 			.padding(.horizontal, Constants.paddingHorizontal)
 		}
+    .navigationBarTitle("")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/c31f92513e6122ea8af17a18dcc4ee1c2cb05e83/ClearKeep/Scenes/Settings/SettingServerView.swift#L114"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>SettingServerView.swift:114</code></td>
    </tr>
  </tbody>
</table>